src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -82,44 +84,32 @@ internal NewRelicTraceExporter(SpanDataSender spanDataSender, TelemetryConfigura
             }
         }
 
-
-        /// <summary>
-        /// Responsible for sending Open Telemetry Spans to New Relic endpoint.
-        /// </summary>
-        /// <param name="batch">Collection of Open Telemetry spans to be sent to New Relic</param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> batch, CancellationToken cancellationToken)
+        /// <inheritdoc />
+        public override ExportResult Export(in Batch<Activity> batch)
         {
-            if (batch == null) return ExportResult.Success;
-
             var nrSpanBatch = ToNewRelicSpanBatch(batch);
 
-            if(nrSpanBatch.Spans.Count == 0)
+            if (nrSpanBatch.Spans.Count == 0)
             {
                 return ExportResult.Success;
             }
 
-            var result = await _spanDataSender.SendDataAsync(nrSpanBatch);
+            Response response = null;
+            Task.Run(async () => response = await _spanDataSender.SendDataAsync(nrSpanBatch)).GetAwaiter().GetResult();
```

Comment:
I am confused by this change.  It feels as if the old code did the same thing without the extra Task.Run.  What's prompting the Task.Run?

Reply:
Prevents potential for deadlock. Using `.GetAwaiter().GetResult();` to invoke an asynchronous method from a synchronous method has some perils. Wrapping in `Task.Run` mitigates this.